here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/IntHandlerForEventHandlers.java

```diff
@@ -57,4 +66,22 @@ public IntHandlerForEventHandlers(final @NotNull INaksha hub) {
       return notImplemented(event);
     }
   }
+
+  private @NotNull Result validateWriteRequest(final @NotNull WriteRequest<?, ?> wr) {
+    Result result = null;
+    for (final WriteOp<?> wOp : wr.queries) {
+      final EventHandler eventHandler = (EventHandler) wOp.feature;
+      // Common Plugin specific validations
+      result = validateWritePluginRequest(eventHandler);
+      if (result instanceof ErrorResult) return result;
+      // TODO : handler specific validations in future, as needed
+      if (eventHandler.getClassName().equals("com.here.naksha.lib.handlers.DefaultStorageHandler")) {
```

Comment:
I think of even slightly better suggestion, that we compare directly using `DefaultStorageHandler.class.getName()` ... reason is, in future if we move the class into different package, this condition will stop working otherwise. So, something like:
```
if (  eventHandler.getClassName().equals(  DefaultStorageHandler.class.getName() ) )
```
This will shout during compilation, if we rename / move the class in future.

Reply:
good idea, thanks!